<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>191</width>
    <height>171</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Быки и коровы 0.1</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <widget class="QPushButton" name="pushButton" >
    <property name="geometry" >
     <rect>
      <x>20</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Новая игра</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>10</y>
      <width>171</width>
      <height>94</height>
     </rect>
    </property>
    <property name="readOnly" >
     <bool>true</bool>
    </property>
    <property name="acceptRichText" >
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit" >
    <property name="geometry" >
     <rect>
      <x>10</x>
      <y>110</y>
      <width>171</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2" >
    <property name="geometry" >
     <rect>
      <x>100</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text" >
     <string>Выход</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>137</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel" >
     <x>95</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
